Week2/arrays.js

```diff
@@ -0,0 +1,15 @@
+// Animals Names
+
+let myFavoriteAnimals;
+
+console.log('The value of myFavoriteAnimals is: undefined then an object \'array\' ');
+
+console.log(myFavoriteAnimals);
+
+myFavoriteAnimals = ['monkey', 'horse', 'turtle'];
+
+console.log(myFavoriteAnimals);
+
+myFavoriteAnimals = myFavoriteAnimals + [', baby pig'];
```

Comment:
It's better to use `myFavoriteAnimals.push('baby pig')'`. This will work but it's not the correct way to do this.